inference-engine/src/transformations/src/transformations/convert_precision.cpp

```diff
@@ -264,6 +266,13 @@ bool fuse_type_to_bucketize(std::shared_ptr<ngraph::Node> & node, ngraph::elemen
 
 bool fuse_type_to_generic_ie(std::shared_ptr<ngraph::Node> & node, ngraph::element::Type to, size_t idx) {
     node->set_output_type(idx, to, node->output(idx).get_partial_shape());
+    auto & rt_info = node->get_rt_info();
```

Comment:
Why do we add to rt inforamtion callback for call a convert precision transformation for Generic IE?

Looks like it is really complex.

Can we in this method just check the type of operation and call a transformation if node has a GenericIE type?

Reply:
No, because of dependency to IE library.